moped-database/views/project_list_view.sql

```diff
@@ -196,14 +196,6 @@ SELECT
     work_activities.task_order_names,
     work_activities.task_order_names_short,
     work_activities.task_orders,
-    'placeholder text'::text AS project_development_status,
```

Comment:
These were mocked out in the project list view, but I have confirmed that they are only going to be used on the component level so I moved them into `componentn_arcgis_online_view`.